src/init.cpp

```diff
@@ -873,6 +873,11 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         return InitError(Untranslated("Cannot set -listen=0 together with -listenonion=1"));
     }
 
+    //if torcontrol given, it needs to be in form of <host>:<port>
```

Comment:
```suggestion
    // if torcontrol given, it needs to be in form of <host>:<port>
```

Reply:
Addressed in 49259637cbd7a59484d8af6c004550ad92d275b2